docs/component_conventions.md

```diff
@@ -164,6 +197,13 @@ Visit the links below for more information:
 * [Official BEM Documentation](https://en.bem.info/methodology/naming-convention/#css-selector-naming-convention)
 * [Guide on BEM naming conventions](https://webdesign.tutsplus.com/articles/an-introduction-to-the-bem-methodology--cms-19403)
 
+### Layout
+
+New components should be built with a bottom margin and no top margin.
+
+It may be desirable for components to have an option to control the size of the bottom margin but there is no standard for this right now. This is a [complex issue](https://github.com/alphagov/govuk_publishing_components/pull/292), a suggested model has been implemented in the [heading component](https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/stylesheets/govuk_publishing_components/components/_heading.scss#L31).
```

Comment:
Non-blocking comment: Is the standard the design system spacing scale?

I imagined being able to pass numbers from the spacing scale and it working on any component but using the helper classes.

https://design-system.service.gov.uk/styles/spacing/

`bottom-spacing: 2`

`<div class="govuk-!-margin-bottom-2"></div>`

Probably a discussion for another PR

Reply:
Ah, good point. That RFC was written before the design system was adopted, I'll have a think about how to reword this bit.

Reply:
Have reworded. Hopefully we'll be able to update this section soon!

Reply:
@nickcolley I'm a bit concerned about giving the option of defining a custom `margin-bottom` to any component (even based on the spacing scale) as it will introduce spacing inconsistencies. I think this is very much the same with allowing custom classes at the component level - the only difference would be that this would affect only the margin property.